nms/app/packages/magmalte/grafana/dashboards/CWFDashboards.js

```diff
@@ -100,8 +100,45 @@ export const CWFSubscriberDBData: GrafanaDBData = {
           description:
             "Should just be 0 for subscribers without an active session and 1 for those with. Not sure what's up with this now.",
         },
+        {
+          title: 'User Growth',
```

Comment:
Hey @sourabh-nanoti I'm a little confused here. Won't this give total number of session creations over time, why does it indicate user growth ?

Reply:
This is what i thought: 
sum(session_start{msisdn=~".*"}) by (msisdn) should give the range vector for each user. Counting those will give how many users CWF has seen. Plotting that over time should give the user growth. 
Does it make sense!

Reply:
I think this metric would go better on the Network page since it's not per-subscriber data. In general that's how the dashboards have been set up. I think introducing metrics where `{msisdn=~".*"}` would be confusing since we have the variable for `$msisdn` but it won't affect this specific graph. Alternatively you could add another dashboard for "Aggregated Subscriber metrics" or something like that. Finally you could even just add a new row where you put these types of metrics and then specify in the name/description of the row that these are aggregated

Reply:
I feel network is more of the message stats and this is more subscriber specific. I agree that it's not per subscriber, but still its relatively close to getting subscriber details. Also using .* does make this panel msisdn selection agnostic. 
I will add a row for aggregate subscriber metrics on the subscribers dashboard. That sounds like a good idea.